src/main/resources/oss/fosslight/repository/LicenseMapper.xml

```diff
@@ -125,8 +125,8 @@
 			, T1.DESCRIPTION
 			, T1.LICENSE_TEXT
 			, T1.ATTRIBUTION
-			, T1.CREATOR
-			, T1.MODIFIER
+			, IFNULL((SELECT USER_NAME FROM T2_USERS WHERE T1.CREATOR = USER_ID), T1.CREATOR) AS CREATOR

```

Comment:
(minor) In my opinion, `SELECT USER_NAME FROM T2_USERS WHERE USER_ID = T1.CREATOR` may better than `SELECT USER_NAME FROM T2_USERS WHERE T1.CREATOR = USER_ID`.

Reply:
I think it would be better to see that the table field is written first in the WHERE clause, but all other queries are in this format, so I resolve this conversation. 😄